### ISSUE TITLE
Add `deleteBatch` method

### ISSUE THREAD

**Issue by xsats** [SatoshiPortal/batcher#4]
I'm sure you must have discussed this, and there is likely a good reason for not including a 'deleteBatch' method at this point, but I think it may be useful in certain scenarios, likely mostly related to testing and error resolution (e.g. when I broke something in #3 ).

**Comment by Kexkey** [comment]
Hi @xsats!  Thanks for testing, it's very appreciated.  Yes, more endpoints are going to be added to the Batcher, particularly functionalities to better "manage" batches and getting some stats.  This version is quite minimal, but it's nevertheless used in production at Bull Bitcoin.

When you talk about deleteBatch, do you mean "cancelling" an ongoing batch?  Or delete a past batch?  Or just removing all the queued requests from the ongoing batch?

**Comment by Kexkey** [comment]
Are you a developer?  If you want to contribute, you are very welcome!

Yes, removing current requests from a queue totally makes sense.  In the branch "dequeueandpay" I added the dequeueAndPay endpoint that just removes a batchRequest from the queue and directly pay to the corresponding address via a standard spend (e.g a client wants to opt out from the next batch and is willing to pay the fees to be paid right away).

**Comment by xsats** [comment]
Yep, we'd like to integrate the batcher in our upcoming platform and I'd love to contribute once I'm a bit more familiar with the project.

dequeuandpay sounds like another fantastic functionality!
